nextflow.config

```diff
@@ -9,6 +9,7 @@
 params {
 
   //MS params
+  input = "data/*.mzML"
```

Comment:
```suggestion
  input = false
```

Reply:
I don't like providing a default, but that's up to you

Reply:
will be removed, it was suggested by the template.